privacyidea/lib/policy.py

```diff
@@ -1298,7 +1324,7 @@ def ui_get_enroll_tokentypes(self, client, logged_in_user):
 def set_policy(name=None, scope=None, action=None, realm=None, resolver=None,
                user=None, time=None, client=None, active=True,
                adminrealm=None, adminuser=None, priority=None, check_all_resolvers=False,
-               conditions=None, pinode=None):
+               conditions=None, pinode=None, user_case_insensitive=True):
```

Comment:
Also default to case-sensitive.

Reply:
[b7bc726](https://github.com/privacyidea/privacyidea/pull/3852/commits/b7bc726f2279a85bad77918536eb77dbabbf0523)